From 17e3978f3ad1c6c4c9aca009a5081c6b0ebb5491 Mon Sep 17 00:00:00 2001
From: Chen Ni <nichen@iscas.ac.cn>
Date: Tue, 14 Jul 2026 16:28:10 +0800
Subject: [PATCH 04974/15333] perf test: Remove duplicate include of
 util/term.h

Remove duplicate inclusion of util/term.h in builtin-test.c to clean up
redundant code.

Signed-off-by: Chen Ni <nichen@iscas.ac.cn>
Reviewed-by: Ian Rogers <irogers@google.com>
Signed-off-by: Namhyung Kim <namhyung@kernel.org>
---
 tools/perf/tests/builtin-test.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tools/perf/tests/builtin-test.c b/tools/perf/tests/builtin-test.c
index 7d18cb6021d3..41dcfbf874f0 100644
--- a/tools/perf/tests/builtin-test.c
+++ b/tools/perf/tests/builtin-test.c
@@ -15,7 +15,6 @@
 #include <string.h>
 
 #include <dirent.h>
-#include "util/term.h"
 #include <linux/kernel.h>
 #include <linux/string.h>
 #include <linux/zalloc.h>
-- 
2.55.0

